tracpro/polls/views.py

```diff
@@ -191,6 +191,31 @@ def get_form_kwargs(self):
 
         def form_valid(self, form):
             form.save()
+
+            # Save the associated Questions for this poll here
+            # now that these polls have been activated for the Org
+            selected_polls = []
+            for poll in form.cleaned_data['polls']:
+                selected_polls.append(poll.name)
+
+            org = self.request.org
+            temba_polls = org.get_temba_client().get_flows(archived=False)
+            temba_polls = {p.uuid: p for p in temba_polls}
+
+            # Create new or update SELECTED Polls to match RapidPro data.
+            for temba_poll in temba_polls.values():
+                if temba_poll.name in selected_polls:
+                    # Sync related Questions, and maintain question order.
+                    temba_questions = temba_poll.rulesets
```

Comment:
This line seems unnecessary, since the next line assigns a different value to the same variable.